src/nncf/onnx/quantization/backend_parameters.py

```diff
@@ -19,11 +19,28 @@ class BackendParameters:
     """
     :param EXTERNAL_DATA_DIR: An absolute path to the directory where the external data
         files are stored. All external data files must be located in the same folder.
+    :param COMPRESS_WEIGHTS: If `True` compresses constant quantized weights by folding
+        `QuantizeLinear` nodes into pre-quantized initializers. If `False`, this transformation
+        is skipped.
```

Comment:
What is the scenario when constants don't need to be compressed? As I understand it, the result will be the same in any case, just the model size is different, right?

Reply:
It seems like this description isn't clear enough. The `COMPRESS_WEIGHTS` parameter (`True` or `False`) determines the format (INT8 or FP) in which the weights of quantized operations will be saved. For a model in ONNX format, this is achieved by applying the `QuantizeLinear` operation to float-precision initializers (i.e., weights).

If you have any suggestions for how we can improve this, please let me know.